<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>codeClass</class>
 <widget class="QMainWindow" name="codeClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="CodeEditorFrame">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4"/>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="goToLineBox">
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="Toolbar" stdset="0">
           <bool>true</bool>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>3</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>3</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Line # </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="sbLineNumber"/>
           </item>
           <item>
            <widget class="QToolButton" name="tbGoToLineNumber">
             <property name="text">
              <string>Go</string>
             </property>
             <property name="icon">
              <iconset resource="../../sharedcode/resources.qrc">
               <normaloff>:/resources/RES/ffficons/control_play_blue.png</normaloff>:/resources/RES/ffficons/control_play_blue.png</iconset>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextBesideIcon</enum>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>403</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QToolButton" name="tbCloseLineNumber">
             <property name="text">
              <string>Close</string>
             </property>
             <property name="icon">
              <iconset resource="../../sharedcode/resources.qrc">
               <normaloff>:/resources/RES/ffficons/cross.png</normaloff>:/resources/RES/ffficons/cross.png</iconset>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="searchBox">
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="Toolbar" stdset="0">
           <bool>true</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="margin">
            <number>3</number>
           </property>
           <property name="spacing">
            <number>3</number>
           </property>
           <item row="0" column="1">
            <widget class="QLineEdit" name="searchText"/>
           </item>
           <item row="0" column="2">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QToolButton" name="tbFindPrev">
               <property name="text">
                <string>Find Previous</string>
               </property>
               <property name="icon">
                <iconset resource="../../sharedcode/resources.qrc">
                 <normaloff>:/resources/RES/ffficons/resultset_previous.png</normaloff>:/resources/RES/ffficons/resultset_previous.png</iconset>
               </property>
               <property name="autoRaise">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="tbFindNext">
               <property name="text">
                <string>Find next</string>
               </property>
               <property name="icon">
                <iconset resource="../../sharedcode/resources.qrc">
                 <normaloff>:/resources/RES/ffficons/resultset_next.png</normaloff>:/resources/RES/ffficons/resultset_next.png</iconset>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextBesideIcon</enum>
               </property>
               <property name="autoRaise">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Ignored</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>5</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QToolButton" name="tbCloseSearch">
               <property name="text">
                <string>Close</string>
               </property>
               <property name="icon">
                <iconset resource="../../sharedcode/resources.qrc">
                 <normaloff>:/resources/RES/ffficons/cross.png</normaloff>:/resources/RES/ffficons/cross.png</iconset>
               </property>
               <property name="autoRaise">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Replace:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="replaceText"/>
           </item>
           <item row="1" column="2">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="spacing">
              <number>2</number>
             </property>
             <item>
              <widget class="QPushButton" name="btReplace">
               <property name="text">
                <string>Replace</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btReplaceAll">
               <property name="text">
                <string>Replace All</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="0" colspan="3">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QCheckBox" name="cbWholeWord">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Whole word only</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="cbCaseSensitive">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Case sensitive</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="cbRegexp">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Use RegExps</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>5</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Search:</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDockWidget" name="classMembersDW">
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Class members</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>1</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="toolButton_2">
          <property name="text">
           <string>Representation</string>
          </property>
          <property name="icon">
           <iconset resource="../../sharedcode/resources.qrc">
            <normaloff>:/resources/RES/ffficons/monitor.png</normaloff>:/resources/RES/ffficons/monitor.png</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>39</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton">
          <property name="text">
           <string>Add event</string>
          </property>
          <property name="icon">
           <iconset resource="../../sharedcode/resources.qrc">
            <normaloff>:/resources/RES/ffficons/clock.png</normaloff>:/resources/RES/ffficons/clock.png</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_3">
          <property name="text">
           <string>Add function</string>
          </property>
          <property name="icon">
           <iconset resource="../../sharedcode/resources.qrc">
            <normaloff>:/resources/RES/ffficons/sum.png</normaloff>:/resources/RES/ffficons/sum.png</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_4">
          <property name="text">
           <string>Add variable</string>
          </property>
          <property name="icon">
           <iconset resource="../../sharedcode/resources.qrc">
            <normaloff>:/resources/RES/ffficons/tag_yellow.png</normaloff>:/resources/RES/ffficons/tag_yellow.png</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QTreeWidget" name="treeWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <column>
        <property name="text">
         <string>Class members</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Information</string>
       </property>
       <property name="icon">
        <iconset resource="../../sharedcode/resources.qrc">
         <normaloff>:/resources/RES/ffficons/information.png</normaloff>:/resources/RES/ffficons/information.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="functionLibraryDW">
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Function library</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolBox" name="toolBox">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="tabSpacing">
        <number>0</number>
       </property>
       <widget class="QWidget" name="fnLibrary">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>87</width>
          <height>397</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Functions library</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string notr="true">To Do:
Functions library</string>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFind"/>
   <addaction name="actionGo_to_line_number"/>
   <addaction name="separator"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="actionCut"/>
   <addaction name="separator"/>
   <addaction name="actionClass_members"/>
   <addaction name="actionFunction_library"/>
   <addaction name="separator"/>
   <addaction name="actionShow_hidden_symbols"/>
   <addaction name="actionShow_tab_lines"/>
   <addaction name="separator"/>
   <addaction name="actionPrint"/>
   <addaction name="actionHelp"/>
  </widget>
  <action name="actionFind">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../sharedcode/resources.qrc">
     <normaloff>:/resources/RES/ffficons/magnifier.png</normaloff>:/resources/RES/ffficons/magnifier.png</iconset>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="../../sharedcode/resources.qrc">
     <normaloff>:/resources/RES/ffficons/page_white_copy.png</normaloff>:/resources/RES/ffficons/page_white_copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="../../sharedcode/resources.qrc">
     <normaloff>:/resources/RES/ffficons/page_white_paste.png</normaloff>:/resources/RES/ffficons/page_white_paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="../../sharedcode/resources.qrc">
     <normaloff>:/resources/RES/ffficons/cut.png</normaloff>:/resources/RES/ffficons/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionClass_members">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../sharedcode/resources.qrc">
     <normaloff>:/resources/RES/ffficons/bricks.png</normaloff>:/resources/RES/ffficons/bricks.png</iconset>
   </property>
   <property name="text">
    <string>Class members</string>
   </property>
  </action>
  <action name="actionFunction_library">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../sharedcode/resources.qrc">
     <normaloff>:/resources/RES/ffficons/book_open.png</normaloff>:/resources/RES/ffficons/book_open.png</iconset>
   </property>
   <property name="text">
    <string>Function library</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="../../sharedcode/resources.qrc">
     <normaloff>:/resources/RES/ffficons/printer.png</normaloff>:/resources/RES/ffficons/printer.png</iconset>
   </property>
   <property name="text">
    <string>print</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="../../sharedcode/resources.qrc">
     <normaloff>:/resources/RES/ffficons/help.png</normaloff>:/resources/RES/ffficons/help.png</iconset>
   </property>
   <property name="text">
    <string>help</string>
   </property>
  </action>
  <action name="actionShow_hidden_symbols">
   <property name="icon">
    <iconset resource="../../sharedcode/resources.qrc">
     <normaloff>:/resources/RES/ffficons/pilcrow.png</normaloff>:/resources/RES/ffficons/pilcrow.png</iconset>
   </property>
   <property name="text">
    <string>Show hidden symbols</string>
   </property>
  </action>
  <action name="actionShow_tab_lines">
   <property name="icon">
    <iconset resource="../../sharedcode/resources.qrc">
     <normaloff>:/resources/RES/ffficons/text_padding_left.png</normaloff>:/resources/RES/ffficons/text_padding_left.png</iconset>
   </property>
   <property name="text">
    <string>Show tab lines</string>
   </property>
  </action>
  <action name="actionGo_to_line_number">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../sharedcode/resources.qrc">
     <normaloff>:/resources/RES/ffficons/page_white_go.png</normaloff>:/resources/RES/ffficons/page_white_go.png</iconset>
   </property>
   <property name="text">
    <string>Go to line number</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../sharedcode/resources.qrc"/>
  <include location="../../sharedcode/resources.qrc"/>
  <include location="../../sharedcode/resources.qrc"/>
  <include location="../../sharedcode/resources.qrc"/>
  <include location="../../sharedcode/resources.qrc"/>
 </resources>
 <connections/>
</ui>
